subsys/input/Kconfig

```diff
@@ -0,0 +1,37 @@
+# Copyright 2023 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig INPUT
+	bool "Input"
+	help
+	  Include input subsystem and drivers in the system config
+
+if INPUT
+
+module = INPUT
+module-str = input
+source "subsys/logging/Kconfig.template.log_config"
+
+config INPUT_INIT_PRIORITY
+	int "Input subsystem and drivers init priority"
+	default 90
+	help
+	  Input subsystem and drivers initialization priority.
+
+config INPUT_THREAD
+	bool "Use a dedicated thread to process input events"
```

Comment:
`depends on MULTITHREADING`

Reply:
I'm still amused that there's a non-multithreading option in an RTOS. :-) added